bugalink-backend-mock/src/data.ts

```diff
@@ -94,6 +94,67 @@ export const sampleTripRequests = [
     price: 1.5,
   },
 ];
+
+export const sampleUser = {
+  id: 1,
+  name: 'Paco',
+  lastName: 'Perez',
+  photo: '/assets/avatar.png',
+};
+
+export const IndividualRides = [
+  {
+    IndividualRideId:1,
+    passenger : {
+      user : {
+        username :'Jesús Marchena',
+        photo: '/assets/avatar.png',
+      }
+    },
+    passenger_routine : {
+      start_location : '88.2231, 88.2324',
+      end_location : '88.2431, 88.2124',
+      days : 'martes y jueves',
+      end_date : '14 de Marzo de 2023',
+      start_time_initial : '8:00',
+      start_time_final : '8:15',
+    },
+    price : 3,
+    ride_status : 'Pending start',
+    acceptation_status : 'Pending Confirmation',
```

Comment:
Should be an enum (In Typescript this could also be a `type` or an `interface`), but not a string